source/server/http/admin.h

```diff
@@ -260,6 +263,7 @@ class AdminImpl : public Admin,
     Stats::ScopePtr scope_;
     Http::ConnectionManagerListenerStats stats_;
   };
+  typedef std::unique_ptr<AdminListener> AdminListenerPtr;
```

Comment:
let's use `using AdminLIstenerPtr =  ...` in new code as it's strictly a superset in functionality.

Reply:
Done.